src/main/java/uk/gov/register/service/ItemValidator.java

```diff
@@ -16,58 +16,59 @@
+        final JsonNode primaryKeyNode = inputEntry.get(registerName.value());
         throwEntryValidationExceptionIfConditionIsFalse(primaryKeyNode == null, inputEntry, "Entry does not contain primary key field '" + registerName + "'");
         validatePrimaryKeyIsNotBlankAssumingItWillAlwaysBeAStringNode(StringUtils.isBlank(primaryKeyNode.textValue()), inputEntry, "Primary key field '" + registerName + "' must have a valid value");
     }
 
-    private void validateFields(JsonNode inputEntry, RegisterMetadata registerMetadata) throws ItemValidationException {
-        Set<String> inputFieldNames = Sets.newHashSet(inputEntry.fieldNames());
-        Set<String> expectedFieldNames = Sets.newHashSet(registerMetadata.getFields());
-        Set<String> unknownFields = Sets.difference(inputFieldNames, expectedFieldNames);
+    private void validateFields(final JsonNode inputEntry, final RegisterMetadata registerMetadata) throws ItemValidationException {
+        final Set<String> inputFieldNames = Sets.newHashSet(inputEntry.fieldNames());
+        final Set<String> expectedFieldNames = Sets.newHashSet(registerMetadata.getFields());
+        final Set<String> unknownFields = Sets.difference(inputFieldNames, expectedFieldNames);
 
         throwEntryValidationExceptionIfConditionIsFalse(!unknownFields.isEmpty(), inputEntry, "Entry contains invalid fields: " + unknownFields.toString());
     }
 
-    private void validateFieldsValue(JsonNode inputEntry, Map<String, Field> fields) throws ItemValidationException {
+    private void validateFieldsValue(final JsonNode inputEntry, final Map<String, Field> fields) throws ItemValidationException {
         inputEntry.fieldNames().forEachRemaining(fieldName -> {
-            Field field = fields.get(fieldName);
-
-            Datatype datatype = field.getDatatype();
-
-            JsonNode fieldValue = inputEntry.get(fieldName);
+            final Field field = fields.get(fieldName);
+            final Datatype datatype = field.getDatatype();
+            final JsonNode fieldValue = inputEntry.get(fieldName);
 
             if (field.getCardinality().equals(Cardinality.MANY)) {
-
                 throwEntryValidationExceptionIfConditionIsFalse(!fieldValue.isArray(), inputEntry, String.format("Field '%s' has cardinality 'n' so the value must be an array of '%s'", fieldName, datatype.getName()));
 
                 fieldValue.elements().forEachRemaining(element -> throwEntryValidationExceptionIfConditionIsFalse(!datatype.isValid(element), inputEntry, String.format("Field '%s' values must be of type '%s'", fieldName, datatype.getName())));
-
             } else {
                 throwEntryValidationExceptionIfConditionIsFalse(!datatype.isValid(fieldValue), inputEntry, String.format("Field '%s' value must be of type '%s'", fieldName, datatype.getName()));
             }
 
+            if ("curie".equals(datatype.getName()) && !fieldValue.textValue().contains(":")) {
```

Comment:
This is a tricky one because ideally this validation would be inside of the `isValid` method of the `CurieDatatype`, but you do not have access to the register field definitions in there as things stand.